inventories/inventory_aws/group_vars/all.yml

```diff
@@ -0,0 +1,4 @@
+cloud: "aws"
+region: "eu-central-1"
+vpc_id: ""
```

Comment:
it is little confusing to have some parts of aws config in `group_vars/all.yml` and here. I would move all the aws specific stuff here. Like for example if they said create_vpc as `false` we need to force them to enter their `vpc_id` and `subnet_id`. If not specified we should fail the role right away.

Reply:
I have added the functionality to verify if _vpc_id_ and _subnet_ids_ variables are set if the user sets vpc_create to false:

> Vault password:
> 
> PLAY [localhost] *********************************************************************************************************************************************
> 
> TASK [cloud_providers : include] *****************************************************************************************************************************
> included: /Users/klins/Documents/yugabyte_exam/fix_branch/aws-ansible/roles/cloud_providers/tasks/aws/infra.yml for localhost
> 
> TASK [cloud_providers : assert] ******************************************************************************************************************************
> fatal: [localhost]: FAILED! => {
>     "assertion": "vpc_id != \"\"",
>     "changed": false,
>     "evaluated_to": false
> }
> 
> MSG:
> 
> make sure to specify vpc_id and subnet_ids inside group_vars/all.yml from inventories folder if vpc_create is set to false
> 
> 
> PLAY RECAP ***************************************************************************************************************************************************
> localhost                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0